### ISSUE TITLE
[bug] `FilterSerializerByOrganization` filters fields without organization

### ISSUE THREAD

**Issue by pandafy** [openwisp/openwisp-users#319]
`FilterSerializerByOrganization` filters queryset of all related fields using the `organization_lookup`. But, it might be possible that a model  does not have relation to `Organization` (not even an indirect one, e.g. [WiFiClient in openwisp-monitoring](https://github.com/openwisp/openwisp-monitoring/pull/382#discussion_r898221497)). 

We should allow to exclude fields from the organization filtering.

**Comment by nemesisdesign** [comment]
@Aryamanz29 do you think this was fixed by https://github.com/openwisp/openwisp-users/commit/670ec9349c27813c130c9421c7b5d7e079230f9e ?

**Comment by Aryamanz29** [comment]
> @Aryamanz29 do you think this was fixed by [670ec93](https://github.com/openwisp/openwisp-users/commit/670ec9349c27813c130c9421c7b5d7e079230f9e) ?

@nemesifier I don't think this issue has been addressed, and it remained unresolved during the development of [multitenant capabilities for the Django Filters feature.](https://github.com/openwisp/openwisp-users/pull/347) This happened because we decided to complete that PR [only for the Wi-Fi session REST API.](https://github.com/openwisp/openwisp-monitoring/issues/374#issuecomment-1559755767) Previously, we were also working on [adding a REST API for Wi-Fi clients](https://github.com/openwisp/openwisp-monitoring/pull/382#discussion_r1182830079), and it was during that time that we encountered this bug.
